spec/features/admin_items_view_spec.rb

```diff
@@ -6,10 +6,12 @@
     @item = FactoryGirl.create(:item, item_status_id: @item_status.id)
     admin = FactoryGirl.create(:admin)
     @category = FactoryGirl.create(:category)
-    visit login_path
-    fill_in("session_email", with: admin.email)
-    fill_in("session_password", with: admin.password)
-    click_button 'Sign in'
+    visit root_path
+    within("#signInModal") do
+      fill_in("session_email", with: admin.email)
+      fill_in("session_password", with: admin.password)
+      click_button 'Sign in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.